spmd/setup.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+import distutils.command.clean
+import glob
+import os
+import shutil
+import subprocess
+from typing import Dict, Union, List
+
+from setuptools import setup, find_namespace_packages
+
+# install: run `python spmd/setup.py install`
+
+# Package name
+package_name = "spmd"
+
+# Version information
+cwd: str = os.path.dirname(os.path.abspath(__file__))
+version_txt: str = os.path.join(cwd, "version.txt")
+with open(version_txt, "r") as f:
+    version: str = f.readline().strip()
+
+try:
+    sha: str = (
+        subprocess.check_output(["git", "rev-parse", "HEAD"], cwd=cwd)
+        .decode("ascii")
+        .strip()
+    )
+except Exception:
+    sha = "Unknown"
+
+if os.getenv("BUILD_VERSION"):
+    version = os.getenv("BUILD_VERSION", version)
```

Comment:
-> `version = os.getenv("BUILD_VERSION")`?
Since we already checked the env var is not None